docs/setting-up-your-environment.md

```diff
@@ -97,13 +116,17 @@ To preview the website in your local browser, make sure you're in the
 `btcinformation.org` directory and run the following command:
 
     make preview
-
+    
 This will compile the site (takes 5 to 10 minutes; see [the speed-up
 instructions](#fast-partial-previews-or-builds)) and then print a
 message like this:
 
     Server address: http://0.0.0.0:4000
     Server running... press ctrl-c to stop.
+    
+On Mac OSX your output may look like this:
+
+    Server address: http://127.0.0.1:4000//    
```

Comment:
It also looks like this to me on linux, so not OSX-specific; consider removing. "A message *like* this" (emphasis mine) should already cover any different server ip or port anyway.

Reply:
I agree that this is unnecessary.